### ISSUE TITLE
about test prompt in inference

### ISSUE THREAD

**Issue by tim20120526** [AILab-CVC/YOLO-World#290]
thanks for your nice work.  i  fine-tune the yolo-world model based on my custom dataset in coco-finetune mode , with the test.json = [ "A", "B", 'C" ].    when i run inference with image_demo.py after training,  the detection results are bad.  I check the code and delete the "[[' ']]" in image_demo.py, then the results are normal. now , i have some questions: 
1.  whats the function of "[[' ']]"  on yolo-world training and testing?

**Comment by wondervictor** [comment]
Firstly, the text json should be `[['A'], ['B'], ['C']]` and each item `['A']` indicates a category(with a set of synonyms).
Secondly, the `[' ']` is a padding category.
